### PR TITLE
Print exceptions on node side

### PR DESCRIPTION
**Pull request by CAMOBAP795**
- example how exception will look like https://github.com/CalConnect/csd-transcription-systems/commit/c7dd0eafbf47332fe752ad86014b03aa45e5d627/checks?check_suite_id=285752229

### CODE REVIEW

**Comment by CAMOBAP795** [comment]
Working on a fix for windows

**Comment by ronaldtse** [comment]
@camobap795 does this work? If so lets merge it.

File: bin/metanorma-pdf.js [hunk 1]
```diff
@@ -7,21 +7,26 @@ const createPdf = async() => {
...
     await page.pdf({
       path: process.argv[3],
       format: 'A4'
     });
   } catch (err) {
-      console.error(err.message);
-      exitCode = 1
+    console.error(err.message);
+    console.error(err.stack);
+    exitCode = 1
```
[the remark this answers is not in the document]
Comment (ronaldtse) [reply bin/metanorma-pdf.js]: @CAMOBAP795 can you fix this?

**Comment by ronaldtse** [comment]
Looks good, feel free to merge once the review comment is fixed.

File: bin/metanorma-pdf.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (CAMOBAP795) [reply bin/metanorma-pdf.js]: @ronaldtse already done

**Pull request merged**